extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -226,22 +271,17 @@ private function doWikiSelector() {
 	 *
 	 * show wiki data
 	 *
-	 * @access public
-	 * @author Krzysztof Krzyżaniak <eloy@wikia.com>
-	 *
-	 * @return nothing
 	 */
 	public function doWikiForm() {
 		global $wgOut, $wgRequest, $wgStylePath, $wgUser;
 		global $wgDevelEnvironment;
 
 		$info = null;
 
-
 		/**
 		 * check maybe something was posted
 		 */
-		if( $wgRequest->wasPosted() ) {
+		if ( $wgRequest->wasPosted() ) {
 			switch( $this->mTab ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)